src/components/modals/EntityExtractor.tsx

```diff
@@ -164,6 +165,29 @@ class EntityExtractor extends React.Component<Props, ComponentState> {
             return false
         })
     }
+
+    // Return list of non-multivalue entity names that have been labelled more than once
+    duplicateEntityNames(extractResponse: CLM.ExtractResponse): string[] {
+        const extractEntities = this.withoutPreBuilts(extractResponse.predictedEntities)
+
+        // Get list of entity ids that are tagged more than once
+        const multiEntityIds = extractEntities.map(pe => pe.entityId)
+            .filter(entityId => {
+                const count = extractEntities.filter(pe => pe.entityId === entityId)
+                return count && count.length > 1
```

Comment:
Wasn't sure why `count && ` was needed.  `filter` should always return arrays so this seems unnecessary.